src/commands/remove/remove_command.py

```diff
@@ -0,0 +1,67 @@
+from logging import getLogger
+from typing import Any, List, Optional
+
+from src.cli import Command
+from src.commands.install.install_command import (
+    EXTERNAL_DEPENDENCY_REFERENCE_DESCRIPTION,
+)
+from src.commands.remove.remove_package import remove_package
+from src.utils import Project, log_color_provider, retrieve_real_package_name
+
+INTERNAL_DEPENDENCY_REFERENCE_DESCRIPTION = (
+    EXTERNAL_DEPENDENCY_REFERENCE_DESCRIPTION
+    + "- `PACKAGE_DIRECTORY_NAME`\n"
+    + "    - `cairo_contracts`, if the package location is `lib/cairo_contracts`"
+)
+
+
+class RemoveCommand(Command):
+    def __init__(self, project: Project) -> None:
+        super().__init__()
+        self._project = project
+
+    @property
+    def name(self) -> str:
+        return "remove"
+
+    @property
+    def description(self) -> str:
+        return "Remove a dependency."
+
+    @property
+    def example(self) -> Optional[str]:
+        return "$ protostar remove cairo-contracts"
+
+    @property
+    def arguments(self) -> List[Command.Argument]:
+        return [
+            Command.Argument(
+                name="package",
+                description=INTERNAL_DEPENDENCY_REFERENCE_DESCRIPTION,
+                type="str",
+                is_required=True,
+                is_positional=True,
+            ),
+        ]
+
+    async def run(self, args):
+        handle_remove_command(args, self._project)
```

Comment:
Do we have task for inlining this methods?

Reply:
The task is created. The TODO pointing to that task is in the `handle_remove_command.py`